java/src/jmri/jmrix/loconet/TranspondingTag.java

```diff
@@ -32,7 +32,7 @@ public TranspondingTag(String systemName, String userName) {
     }
 
     @Override
-    public String toString(){
+    public String toReportString(){
```

Comment:
If I revert this change, JMRI will not compile since this PR makes AbstractNamedBean.toString() final and therefore TranspondingTag cannot override toString().

The purpose of this PR is to show the consequences of making `AbstractNamedBean.toString()` final.

I can revert the change if the purpose of doing that is to get the compile error.

Reply:
There are two problems here:

1. This isn’t the right change.  The intention is that the toString in TranspondingTag eventually will just be removed.
2. We need to give users time to adjust ( scripts in particular).  

In another PR, I added the proper deprecation warnings to the toString method in TranspondingTag.  Our normal cycle for removing those is at least 2 years.

Reply:
The change is now reverted.